### ISSUE TITLE
Worldgen crash Using Koresample 1.2.5 and Dendrology 1.3.0 [forge 10.13.2.1352]

### ISSUE THREAD

**Issue by ate0ate** [MinecraftModArchive/Dendrology#30]
I posted this on the KoreSample github, but perhaps it should be here instead...

After wandering around a new SSP world and generating new chunks, I got this crash out of the blue. I do indeed have Ancient Trees config set to generate trees in the overworld very rarely. So rarely I've yet to encounter one. With that being said, I am unsure if this is a weird one off crash or something that would happen every time an Ancient tree generates. I am using fastcraft, which is also mentioned in the crash report. I also use many mods, so the area attempting to generate does happen to be a Biomes O Plenty biome with an Applied Energistics 2 Meteor right in the vicinity, if either of those things even make any difference. Thank you for the great mod and I apologize if this issue turns out not to be Koresample's fault.

http://pastebin.com/zPBe8j67

**Comment by ScottKillen** [comment]
Posting here is fine! :grinning: 

Any idea how a slab was generated during worldgen? The error is happening because the renderer is trying to get a texture for a slab subblock that does not exist. Neither Dendrology or KoreSample places slabs during worldgen.

If you have any details, that would be helpful...otherwise I'm not sure how to duplicate this bug to figure out what happened...

**Comment by ScottKillen** [comment]
And...for the record, Minecraft crash reports are *never* as good as forge logs. :smiling_imp:

**Comment by ate0ate** [comment]
Hmm, sorry about that. I have no idea why slabs would be trying to generate really. It may be possible that witchery or Biomes O Plenty attempted to generate a village using one of your slabs. Both use surrounding tree types to determine which wood to use in village structures. Witchery's recent village generation changes in particular would be my only guess. I unfortunately wiped my logs so I cannot post them, but I will certainly do so if it occurs again.

The crash made the world unplayable, but since it was fairly new, I simply started a new one so I can't load it and produce the crash again either. I really dropped the ball on this one. I did disable ancient tree world generation and have not encountered this crash again but I will re-enable it and see what happens. I'll report back if I experience the same crash. This time with the proper logs:P
Suppose I'll close this for now...

Thanks for your time!
